versions.lock

```diff
@@ -37,34 +37,36 @@ com.sun.jersey:jersey-json:1.9 (5 constraints: 945f2f90)
 com.sun.jersey:jersey-server:1.9 (4 constraints: ef373c01)
 com.sun.jersey.contribs:jersey-guice:1.9 (4 constraints: 65529ed7)
 com.sun.xml.bind:jaxb-impl:2.2.3-1 (1 constraints: 330c2404)
+com.tdunning:json:1.8 (1 constraints: 910b72db)
 com.thoughtworks.paranamer:paranamer:2.8 (3 constraints: 742d4cb1)
 com.twitter:chill-java:0.9.3 (2 constraints: a716716f)
 com.twitter:chill_2.11:0.9.3 (2 constraints: 121b92c3)
-com.twitter:parquet-hadoop-bundle:1.6.0 (3 constraints: 7c262424)
+com.twitter:parquet-hadoop-bundle:1.6.0 (2 constraints: 061b4d93)
 com.univocity:univocity-parsers:2.7.3 (1 constraints: c40ccb27)
+com.zaxxer:HikariCP:2.5.1 (1 constraints: 390d7120)
```

Comment:
Do we know what this is and whether it is needed?

Reply:
Looks like this is a JDBC connection pool. Since we are using the thrift API, we probably don't need it in the Spark runtime module, but it's probably necessary for running tests that use a built-in metastore.

Reply:
Let me take a try if it still can be worked after removing this dependencies.

Reply:
Seems not necessary to add, I've already excluded this dependency.